packages/cloud_firestore/lib/src/firestore.dart

```diff
@@ -32,6 +36,10 @@ class Firestore {
           this,
         );
         _documentObservers[call.arguments['handle']].add(snapshot);
+      } else if (call.method == 'DoTransaction') {
+        final int transactionId = call.arguments['transactionId'];
+        return _transactionHandlers[transactionId](
+            new Transaction(transactionId));
```

Comment:
This is a weird indentation, consider a trailing comma or assign `final Transaction transaction = new Transaction(transactionId);` on the previous line

Reply:
Used trailing comma.